src/main/java/dk/cngroup/lentils/entity/Team.java

```diff
@@ -10,13 +18,17 @@
     @Id
     @GeneratedValue
     @Column(name = "id")
-    private long id;
+    private Long id;
 
     @Column(name = "name", nullable = false, length = 50)
+    @NotEmpty
     private String name;
 
     @Column(name = "num_of_members", length = 1)
-    private int numOfMembers;
+    @NotNull(message = "must not be empty")
+    @Min(value = 1, message = "You must enter a number greater or equal to 1")
+    @Max(value = 9, message = "You must enter a number less or equal to 9")
```

Comment:
I kdyz tak velke tymy tezko budou, neomezoval bych explicitne na 9, Nebo to byl nejaky pozadavek?

Reply:
Aha, ja myslel, ze je to implicitne omezene na velikost integeru. Tak v tom pripade OK :)